src/EditorFeatures/Core/ExternalAccess/VSTypeScript/VSTypeScriptDiagnosticService.cs

```diff
@@ -3,25 +3,21 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Collections.Immutable;
+using System.Composition;
 using System.Threading;
 using System.Threading.Tasks;
-using System.Composition;
-using System.Collections.Immutable;
-using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript.Api;
-using Microsoft.CodeAnalysis.Diagnostics;
-using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Host.Mef;
 using Roslyn.Utilities;
 
 namespace Microsoft.CodeAnalysis.ExternalAccess.VSTypeScript;
 
 [Export(typeof(IVSTypeScriptDiagnosticService)), Shared]
 [method: ImportingConstructor]
 [method: Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
-internal sealed class VSTypeScriptDiagnosticService(IDiagnosticService service) : IVSTypeScriptDiagnosticService
+internal sealed class VSTypeScriptDiagnosticService() : IVSTypeScriptDiagnosticService
```

Comment:
this is the only place that actually takes the events from IDiagnosticService and forwards them along.  But it's only for "push style" diagnostics, which we've moved off of, and which TS also definitely does not want.  @MariaSolOs for awareness.  this should go nicely with yoru move to LSP pull diags.